src/TaskRunner/Commands/BuildCommands.php

```diff
@@ -65,8 +65,8 @@ public function buildDist(array $options = [
         'dist-root' => InputOption::VALUE_REQUIRED,
         'keep' => InputOption::VALUE_REQUIRED,
         'remove' => InputOption::VALUE_REQUIRED,
-        'tag' => InputOption::VALUE_OPTIONAL,
-        'sha' => InputOption::VALUE_OPTIONAL,
+        'tag' => InputOption::VALUE_REQUIRED,
+        'sha' => InputOption::VALUE_REQUIRED,
```

Comment:
Why not keep them optional and set them to null or 'dev' if not given?
So locally we can just test the using: `./vendor/bin/run toolkit:build-dist`

Reply:
Hi @vever001 ,
The InputOption::VALUE_REQUIRED only tell symfony that this options requires a value rather then accepting a optional value (which is not correct)
If is set to VALUE_OPTIONAL, this is possible to do
```
./vendor/bin/run toolkit:build-dist --tag
```
If is VALUE_REQUIRE, it is required to have a value, otherwise it fails
```
./vendor/bin/run toolkit:build-dist --tag=0.0.1
```